docs/topics/class-based-views/index.txt

```diff
@@ -101,7 +102,8 @@ preferable to ask the API when the most recent book was published.
 We map the URL to book list view in the URLconf::
 
     from django.urls import path
-    from books.views import BookListView
+
+    from .views import BookListView
```

Comment:
Leave alone. This looks like a root URLConf.

Reply:
ok